localizations/zh_CN.json

```
@@ -160,71 +160,71 @@
     "Apply Logo": "应用 Logo",
     "Apply Lora to outputs rather than inputs when possible (experimental)": "尽可能将 LoRA 模型应用于输出而非输入(实验性选项)",
     "Apply Style": "应用风格",
-    "Apply and restart UI": "应用并重启用户界面",
-    "Apply block weight from text": "将手动输入的权重应用于滑动条",
+    "Apply and restart UI": "应用更改并重载前端",
+    "Apply block weight from text": "从文本框应用分块权重值",
     "Apply changes (Reload UI)": "应用设置（重启 WebUI）",
     "Apply color correction to img2img results to match original colors.": "对图生图结果应用颜色校正以匹配原始颜色",
     "Apply settings": "保存设置",
-    "Apply to": "应用于",
-    "Apply transfer control when loading models": "加载模型时应用传输控制",
+    "Apply to": "应用到",
+    "Apply transfer control when loading models": "加载模型时应用控制转移",
     "Artist Tags": "画师标签(Tags)",
     "Artists to study": "艺术家图库",
     "Attention grabber": "随机关键词吸引注意力",
     "Author": "作者",
     "Author of this model": "此模型的作者",
     "Auto TLS-HTTPS": "自动 TLS-HTTPS",
-    "Auto focal point crop": "自动焦点裁切",
-    "Automatic": "自动（仅加载与模型同名的 VAE，未找到时不加载）",
+    "Auto focal point crop": "自动面部焦点剪裁",
+    "Automatic": "自动识别",
     "Automatically update your prompt with interesting modifiers. (Runs slowly the first time)": "使用有趣的修饰符自动更新你的提示词。（第一次运行会比较慢）",
-    "Available": "可用",
+    "Available": "可下载",
     "Average image color": "平均图像颜色",
     "Background": "背景",
-    "Bake in VAE": "整合入模型的 VAE",
+    "Bake in VAE": "嵌入 VAE 模型",
     "Batch": "批量处理",
     "Batch Process": "批量处理",
-    "Batch Size": "每批数量(Batch Size)",
-    "Batch count": "生成批次",
-    "Batch from Directory": "从目录进行批量处理",
-    "Batch from directory": "批量操作",
-    "Batch size": "每批数量",
+    "Batch Size": "单批数量",
+    "Batch count": "总批次数",
+    "Batch from Directory": "批量处理文件夹",
+    "Batch from directory": "批量处理文件夹",
```

```suggestion
    "Batch from Directory": "批量处理目录",
    "Batch from directory": "批量处理目录",
```
Directory - 目录  
Folder - 文件夹